lib/modules/commands/actions.rb

```diff
@@ -1,144 +1,443 @@
+        response = Database::Response.where(key: 'adverb')
+                                      .all.sample.response
+        event << "#{event.user.display_name} #{response} pokes #{thing}"
+        event.message.delete
+      end
+
+      command(:pet,
+              description: 'Pet other users!',
+              usage: "#{BOT.prefix}pet @user/something") do |event, *thing|
+              next event.respond "I need the `playful` role and I need to be able to delete messages" unless
+              event.bot.profile.on(event.server).roles.map {|x| x.name }.join.include? 'playful'
+        thing = thing.join(' ')
+        response = Database::Response.where(key: 'adverb')
+                                      .all.sample.response
+        event << "#{event.user.display_name} #{response} pets #{thing}"
+        event.message.delete
+      end
+
+      command(:spray,
+              description: 'Spray other users with various, deadly, weaponry!',
+              usage: "#{BOT.prefix}spray @user/something") do |event, *thing|
+              next event.respond "I need the `playful` role and I need to be able to delete messages" unless
+              event.bot.profile.on(event.server).roles.map {|x| x.name }.join.include? 'playful'
+        thing = thing.join(' ')
+        response = Database::Response.where(key: 'watercontainer')
+                                              .all.sample.response
+        event << "#{event.user.display_name} sprays #{thing}"\
+                  " with a #{response}"
+        event.message.delete
+      end
+
+      command(:stare,
+              description: 'Stare at people, maybe senpai will notice',
+              usage: "#{BOT.prefix}stare @user/something") do |event, *thing|
+              next event.respond "I need the `playful` role and I need to be able to delete messages" unless
+              event.bot.profile.on(event.server).roles.map {|x| x.name }.join.include? 'playful'
+        thing = thing.join(' ')
+        event << "#{event.user.display_name} stares at #{thing}"
+        event.message.delete
+      end
+
+      command(:bite,
+              description: 'Bite other users!',
+              usage: "#{BOT.prefix}bite @user/something") do |event, *thing|
+              next event.respond "I need the `playful` role and I need to be able to delete messages" unless
+              event.bot.profile.on(event.server).roles.map {|x| x.name }.join.include? 'playful'
+        thing = thing.join(' ')
+        response = Database::Response.where(key: 'adverb')
+                                      .all.sample.response
+        event << "#{event.user.display_name} #{response} bites #{thing}"
+        event.message.delete
+      end
+
+      command(:bad,
+              description: 'Throws people into timeout',
+              usage: "#{BOT.prefix}bad @user/something") do |event, *message|
+              next event.respond "I need the `playful` role and I need to be able to delete messages" unless

```

Comment:
On all of these patterns (`next even.repond`) should all follow

```ruby
next 'I need the {role} role' unless {condition}
```

- Use single quotes
- `event.respond` is redundant and not needed